ArcGISRuntimeSDKQt_CppSamples/Search/OfflineGeocode/README.metadata.json

```diff
@@ -6,20 +6,8 @@
             "path": "~/ArcGIS/Runtime/Data/tpkx/streetmap_SD.tpkx"
         },
         {
-            "itemId": "c88c170e18f74740a0e4c660a5ff51c4",
+            "itemId": "3424d442ebe54f3cbf34462382d3aebe",
             "path": "~/ArcGIS/Runtime/Data/Locators/SanDiegoStreetAddress/SanDiego_StreetAddress.loc"
```

Comment:
This might be worth testing out the following scenario:

* run the old viewer and download the data for offline geocode sample
* run the new viewer with these changes, and open offline geocode - it will find the old locator and not prompt to download the new one

This means that samples will continue to run against the older data if it has already been downloaded. It might be worth changing the file path slightly so that the new locators go to a new location. This will ensure the sample viewer downloads the new data, while old viewers can continue running against the old data

Reply:
Good point. The 100.12 implementation of the offline geocode in the sample viewer doesn't work if the old locator is downloaded to that location. Any thoughts on what the new path should be? I'm thinking `"~/ArcGIS/Runtime/Data/Locators/SanDiegoStreetAddressLocator/SanDiego_StreetAddress.loc"`. Appending "Pro" to the end instead wouldn't make sense because they were both made in Pro. The name for these new locators is the "eagle locator" but if we put that in, people may be misled if they expect a map with eagle locations on it.

Reply:
Your suggestion works for me